core/aws-lsp-fqn/src/browser/FqnWorkerPool.ts

```diff
@@ -0,0 +1,29 @@
+import { CommonFqnWorkerPool } from '../common/CommonFqnWorkerPool'
+import { ExtractorResult, FqnExtractorInput, WorkerPoolConfig } from '../common/types'
+
+// TODO: implement logic for browser/webworker environment
+export class FqnWorkerPool extends CommonFqnWorkerPool {
```

Comment:
nit: it would look a bit cleaner here if `FqnWorkerPool` just implemented `IFqnWorkerPool`. This stub does not need common logic for any behavior.